### DISCUSSION TITLE
Is there any feature like JavaScript Array push() Method ?

### DISCUSSION THREAD

**Discussion by ramesh-khatri** [getgauge/taiko#1776] [General]
I want to push the text to the array
let myArray = []
(await $('table> tbody> tr> td:nth-child(2)').elements()).forEach(async (ele) => {
       let myData = await ele.text()
       myArray.push(myData.trim())
    });
But, I was not able to do so. I was not able to do the same using evaluate API also. Is there any way to do so?

**Comment by zabil** [comment]
You can refer https://docs.taiko.dev/working_with_element_lists/ for more info how to loop through elements and read values.

As mentioned in the documentation above

> You cannot use callback methods on the list like forEach(...) as they are not async aware.

**Reply by ramesh-khatri** [reply]
Thank you so much.
